04.ls/ls.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+FIND_FILES = Dir.glob('*')
+SPLIT_NUMBER = 3
+
+def splitting_array
+  max_length = FIND_FILES.length / SPLIT_NUMBER
+  FIND_FILES.length % SPLIT_NUMBER != 0 ? max_length += 1 : max_length
```

Comment:
```suggestion
  max_length = FIND_FILES.length % SPLIT_NUMBER != 0 ? max_length + 1 : max_length
```

こう書いた方が `max_length` が書き替えられたことを把握しやすいです。

ただし、商を小数点以下まで出して切り上げする、という形にすると1行で済みそうです。
また、変数の再代入を防ぐ、という効果もあります。
[プログラミング初心者は変数の再代入をなるべく避けよう \| FBC](https://bootcamp.fjord.jp/pages/avoid-re-assign)